src/main/java/frc/team4909/robot/RobotConstants.java

```diff
@@ -4,27 +4,25 @@
 
 public class RobotConstants {
   /* Operator Input */
-  public static final int driverGamepadPort = 0;
   public static final double driverGamepadSensitivity = 0.6; // Value selected from 2018 code
-  public static final int driverGamepadDeadzone = 0; // Value selected from 2018 code
+  public static final double driverGamepadDeadzone = 0.05; // Value selected from 2018 code
 
-  public static final int manipulatorGamepadPort = 1;
   public static final double manipulatorGamepadSensitivity = 0.6; // Value selected from 2018 code
-  public static final int manipulatorGamepadDeadzone = 0; // Value selected from 2018 code
+  public static final double manipulatorGamepadDeadzone = 0.05; // Value selected from 2018 code
 
   /* Drivetrain */
   public static final double speedMultiplier = .7;  
 
   /* Intake */
   public static final double cargoIntakeInSpeed = 1.0;
-  public static final double cargoIntakeHoldSpeed = 0;
+  public static final double cargoIntakeHoldSpeed = 0.0625;
   public static final double cargoIntakeOutSpeed = -1.0;
 
-  public static final double cargoIntakeCurrentLimit = 20;
+  public static final double cargoIntakeCurrentLimit = 0;
 
   /* LIDAR */
   public static final int calibrationOffset = -2;
-  public static final double lidarLimit = 20; // cm
+  public static final double lidarLimit = 30; // cm
```

Comment:
what does this number even mean?

Reply:
@ShashJar comments please

Reply:
Lidar sensor isn't placed at the exact edge of the robot, therefore we need to account for the width of the bumper/space between bumper and lidar - we measured this to be ~9 cm.